docs/docs/providers/gcs.md

```diff
@@ -17,7 +17,38 @@ Access can be given at the bucket level or object level as those allowed to be m
 ** NOTE ** : Object level access can only be provide for objects belonging to a **Fine-granied** bucket. For objects belonging to the **Uniform-level**, permissions are only given at the bucket level.
 
 ## Config
-TODO
+```yaml
+type: gcs
+urn: sample-URN
+credentials:
+  service_account_key: <base64 encoded service account key json>
+  resource_name: projects/<gcs-project-id>
+  - type: bucket
+    policy:
+      id: my-first-policy
+      version: 1
+    roles:
+      - id: READER
+        name: 'Grants permission to list a bucket contents and read bucket metadata, excluding IAM policies'
```

Comment:
`name` should be the role's display name, and this can go into `description` field

Reply:
pushed in the latest commit